File: NewsPortal/NewsPortal/Database/ArticlePayment.cs
using System;

namespace NewsPortal.WebAPI.Database
{
    public class ArticlePayment
    {
        public int Id { get; set; }
        public double Amount { get; set; }
        public string TansactionNumber { get; set; }
        public DateTime TransactionDate { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }

    }
}

File: NewsPortal/NewsPortalMobile/newsportal/lib/models/ArticlePayment.dart
import 'dart:convert';

class ArticlePayment {
  final double Amount;
  final String TansactionNumber;
  final DateTime TransactionDate;
  final int? UserId;
  final int? ArticleId;
  ArticlePayment({
    required this.Amount,
    required this.TansactionNumber,
    required this.TransactionDate,
    required this.UserId,
    required this.ArticleId

  });

  factory ArticlePayment.fromJson(Map<String, dynamic> json) {
    return ArticlePayment(
      Amount: json['amount'],
      TansactionNumber: json['tansactionNumber'],
      UserId: json['userId'],
      TransactionDate: DateTime.parse(json['transactionDate']),
      ArticleId: json['articleId'],

    );
  }

  Map<String, dynamic> toJson() => {
        "amount": Amount,
        "tansactionNumber": TansactionNumber,
        "userId": UserId,
        "transactionDate": TransactionDate.toString(),
       'articleId':ArticleId
      };
}
